.rubocop.yml

```diff
@@ -38,6 +39,13 @@ Style/DefWithParentheses:
   Enabled: false
 Style/FrozenStringLiteralComment:
   EnforcedStyle: never
+Style/OptionalBooleanParameter:
+  Enabled: false
+Style/OptionalBooleanParameter:
+  Enabled: true
+Style/OptionalBooleanParameter:
+  Enabled: true
+  EnforcedStyle: keyword_arguments  
```

Comment:
- [ ] You shouldn't disable Rubocop cause it helps you to write clean code so please make sure to remove it and keeps the settings as it was